src/billo.php

```diff
@@ -32,6 +32,18 @@ function distributeMessage($listsConfig, $mailSender, $id, $message) {
     $mailSender->Subject =
       '['.$listsConfig->getListName($list->getEmail()).'] '.$message->subject;
 
+    $members = $listsConfig->getMembers($list->getEmail());                                                                                                                                         
+     // Check at least one of the senders ("from") is subscribed to the list                                                                     
+     $subscribed = false;                                                                                                                        
+     foreach($message->getHeader('from')->getAddressList() as $from) {
+      if ($listsConfig->isMember($from->getEmail(), $members)) {
```

Comment:
You need to pass the list name to `isMember` not the members themselves.
```suggestion
      if ($listsConfig->isMember($from->getEmail(), $list->getEmail())) {
```